components/organisms/o-product-details.vue

```diff
@@ -1,53 +1,85 @@
 <template>
-  <div class="o-product-details">
+  <div class="o-product-details" itemprop="offers" itemscope itemtype="http://schema.org/Offer">
+    <meta itemprop="priceCurrency" :content="$store.state.storeView.i18n.currencyCode">
+    <meta itemprop="price" :content="parseFloat(product.price_incl_tax).toFixed(2)">
+    <meta itemprop="availability" :content="availability">
+    <meta itemprop="url" :content="product.url_path">
     <MProductGallery
       :offline-image="offlineImage"
       :gallery="gallery"
       :configuration="productConfiguration"
     />
-    <div class="o-product-details__description">
+    <div class="description">
       <SfSticky>
-        <MProductShortInfo :product="product" :custom-options="productCustomOptions" />
+        <MProductShortInfo
+          :product="product"
+          :custom-options="productCustomOptions"
+          :reviews="reviews"
+        />
         <ATextAction
-          class="o-product-details__text-action"
+          v-if="sizeOption"
+          class="text-action"
           text="Size guide"
```

Comment:
To trzeba wrzucić w funkcję do tłumaczeń

Reply:
done